lib/project_types/theme/messages/messages.rb

```diff
@@ -21,16 +21,35 @@ module Messages
           },
         },
         forms: {
+          ask_password: "Password:",
+          ask_store: "Store domain:",
           create: {
-            ask_password: "Password:",
-            ask_store: "Store domain:",
             ask_title: "Title:",
-            errors: "%s can't be blank",
             private_app: <<~APP,
               To create a new theme, Shopify App CLI needs to connect with a private app installed on your store. Visit {{underline:%s/admin/apps/private}} to create a new API key and password, or retrieve an existing password.
               If you create a new private app, ensure that it has Read and Write Theme access.",
             APP
           },
+          errors: "%s can't be blank",
+          pull: {
+            private_app: <<~APP,
+              To fetch your existing themes, Shopify App CLI needs to connect with your store. Visit {{underline:%s/admin/apps/private}} to create a new API key and password, or retrieve any existing password.
+              If you create a new private app, ensure that it has Read and Write Theme access.",
+            APP
+          },
+        },
+        pull: {
+          help: <<~HELP,
+            {{command:%s pull}}: Connects an existing theme in your store to Shopify App CLI. Creates a local copy of theme files on Shopify.
+              Usage: {{command:%s pull}}
+              Options:
+                {{command:--store=MYSHOPIFYDOMAIN}} Store URL. Must be an existing store with private apps enabled.
+                {{command:--password=PASSWORD}} Private app password. App must have Read and Write Theme access.
+                {{command:--themeid=THEMEID}} Theme ID. Must be an existing theme on your store.
+          HELP
+          pull: "Pulling theme...",
+          failed: "Couldn't pull from store",
+          pulled: "{{green:%s}} was pulled from {{underline:%s}} to {{green:%s}}",
```

Comment:
```suggestion
          pull: "Pulling theme files...",
          failed: "Couldn't pull theme files from store",
          pulled: "{{green:%s}} files were pulled from {{underline:%s}} to {{green:%s}}",
```

Just adding a smidge more detail.

Just to double check: this last line is structured this way, correct?: "{THEMEID} was pulled from {STORE} to {/local/directory/structure}"

Reply:
It's the name of the theme, as opposed to the ID, but yes, same deal!